pkg/skaffold/util/util.go

```diff
@@ -66,25 +64,27 @@ func StrSliceContains(sl []string, s string) bool {
 
 // ExpandPathsGlob expands paths according to filepath.Glob patterns
 // Returns a list of unique files that match the glob patterns passed in.
-func ExpandPathsGlob(paths []string) ([]string, error) {
+func ExpandPathsGlob(workingDir string, paths []string) ([]string, error) {
```

Comment:
Was this code just wrong before without the workingDir?

Reply:
No. It was using afero that implicitly sets a root dir